Tasks/Common/sdkutils.ts

```diff
@@ -58,7 +58,7 @@ export abstract class SdkUtils {
     public static getTempLocation(): string {
         let tempDirectory = tl.getVariable(this.agentTempDirectoryVariable)
         if (!tempDirectory) {
-            tempDirectory = process.env.TEMP
+            tempDirectory = process.env.TEMP || ''
```

Comment:
Is '' a safe fallback (will that cause errors)? Should the code do something like throw Error or find an alternative location if process.env.TEMP is not set?

Secondary: what if process.env.TEMP is also set to ''?

Reply:
This is a safe default because it will be set to working directory, which is the safest default possible I think. to answer secondly, it will be set to working directory which is fine.